### PR TITLE
Fix/nlp geolocation change

### PR DESCRIPTION
**Pull request by sudan45**
## Changes

* change of geo location in analysis model and auto assisted taggings

Mention related users here if any.

## This PR doesn't introduce any:

- [ ] temporary files, auto-generated files or secret keys
- [ ] n+1 queries
- [ ] flake8 issues
- [ ] `print`
- [ ] typos
- [ ] unwanted comments

## This PR contains valid:

- [ ] tests
- [ ] permission checks (tests here too)
- [ ] translations

### CODE REVIEW

File: apps/analysis/admin.py [hunk 1]
```diff
@@ -32,3 +34,13 @@ class AnalyticalStatementAdmin(admin.ModelAdmin):
 @admin.register(DiscardedEntry)
 class DiscardedEntryAdmin(admin.ModelAdmin):
     pass
+
+
+@admin.register(AnalyticalStatementGeoEntry)
+class AnalyticalStatementGeoEntryAdmin(admin.ModelAdmin):
+    pass
```
Comment (thenav56) [inline apps/analysis/admin.py]: Let's add autocomplete fields

File: apps/analysis/admin.py [hunk 2]
```diff
@@ -32,3 +34,13 @@ class AnalyticalStatementAdmin(admin.ModelAdmin):
...
+
+
+@admin.register(AnalyticalStatementGeoTask)
+class AnalyticalStatementGeoTaskAdmin(admin.ModelAdmin):
+    pass
```
Comment (thenav56) [inline apps/analysis/admin.py]: Let's add autocomplete fields

File: apps/analysis/tests/test_mutations.py [hunk 3]
```diff
@@ -759,21 +759,17 @@ def _query_check(_id):
         SAMPLE_GEO_DATA_RESPONSE = [
             {
                 'entry_id': str(entry.id),
-                'entities': [
+                'locatiion': [
```
Comment (thenav56) [inline apps/analysis/tests/test_mutations.py]:
```suggestion
                'location': [
```

**Review by thenav56 — CHANGES_REQUESTED** [review]
Looks good. Few comments

File: apps/analysis/admin.py (hunk 1 above)
Comment (sudan45) [reply apps/analysis/admin.py]: just use for the test 
i will remove this code from admin.oy

File: apps/deepl_integration/views.py [hunk 4]
```diff
@@ -24,6 +24,7 @@ class BaseCallbackView(views.APIView):
     permission_classes = [permissions.AllowAny]
 
     def post(self, request, **_):
+        print(request.data)
```
Comment (thenav56) [inline apps/deepl_integration/views.py]: hmmmmm

**Review by thenav56 — CHANGES_REQUESTED** [review]

File: apps/deepl_integration/views.py (hunk 4 above)
Comment (Rup-Narayan-Rajbanshi) [reply apps/deepl_integration/views.py]: hmmmmmmmmm.......

**Review by Rup-Narayan-Rajbanshi — APPROVED** [review]

**Review by Rup-Narayan-Rajbanshi — APPROVED** [review]

**Review by thenav56 — APPROVED** [review]

**Pull request merged**